requirements/main.txt

```diff
@@ -790,6 +790,10 @@ html5lib==1.1 \
     --hash=sha256:0d78f8fde1c230e99fe37986a60526d7049ed4bf8a9fadbad5f00e22e58e041d \
     --hash=sha256:b2e5b40261e20f354d198eae92afc10d750afb487ed5e50f9c4eaf07c184146f
     # via -r requirements/main.in
+humanize==2.5.0 \
```

Comment:
This is a pretty old version of the library. 4.6.0 is latest, and includes type hints that should solve the lint failing.
Would you mind removing this line and running `make deps` again to get the current version?

Reply:
Ready @miketheman thanks!